### ISSUE TITLE
Learning Bronchiole-Sensitive Airway Segmentation CNNs by Feature Recalibration and Attention Distillation

### ISSUE THREAD

**Issue by 3D-ICE-arXiv** [3D-ICE-arXiv/Papers#25]
Shanghai Jiao Tong University
Guizhou University, Guiyang, China

**■どんなもの？**
気管（Airway）の抽出はタスクは、気管の細く、長い性質とその背景の情報により困難なタスクである。
またその枝分かれした構造も困難なタスクの要因である。
このようなタスク細気管支(bronchiole)対して、細い領域に注目したモデルを提案。

**■先行研究と比べてどこがすごい？**
thresholding, region growing, and tubular structure enhancing.
these methods often failed to extract peripheral bronchioles and produced leakage outside wall.

First, severe class imbalance between airways and background poses a threat to the training of 3-D CNNs.
Most of the current CNNs heavily rely on airway ground-truth as supervisory signals.
Unlike bulky or spheroid-like organs (e.g., liver and kidney), tree-like airways are thin, tenuous, and divergent. 
It is difficult to train deep models using such sparse and scattered target labels.

Second, the characteristics of airways require the model to utilize both global-scale and local-scale context knowledge to perceive the main body (trachea and main bronchus) and limbs (peripheral bronchi).
Previous models used 2 or 3 pooling layers and the coarsest resolution features provide limited long-range context. If more layers are simply piled up, the increased parameters may cause over-fitting due to inadequate training data.

To address these two concerns, we present a CNNs-based airway segmentation method with high sensitivity to peripheral bronchioles. Our contributions are threefold: 
1) The recalibration module is proposed to maximally utilize the learned features.
2) The attention distillation module is introduced to reinforce representation learning of airway structure.
3) With extensive experiments on 90 clinical 3-D CT scans, our method achieved superior sensitivity to extraction of thin airways and maintained competitive overall segmentation performance.

**■技術や手法のキモはどこ？**
feature recalibration moduleをまず用意し、Learned featureを作成する。
attention distillationをAirway specific representation learningを補強するために導入

**■どうやって有効だと検証した？**
90枚の３D thoracic CTデータを使用。
20枚を[LIDC](https://github.com/3D-ICE-arXiv/Papers/issues/26)から
70枚を[Exact'09](https://github.com/3D-ICE-arXiv/Papers/issues/27)から取得
従来研究との精度を差を比較することでモデルの有効性を検証

**■議論はある？**
特になし。

**■次に読むべき論文は？**
意外と行われている気管抽出タスク。

several automatic airway segmentation methods have been proposed [11,12,19,21]
Recent progress of convolutional neural networks (CNNs) [4,5,14,17] has spawned research on airway segmentation using CNNs [3,7–9,13,15,18,20,22,25].
optimized tracking way [13,25]
sliding window way [8].
recurrent convolution[20]
Graph neural networks [9,18]

![image](https://user-images.githubusercontent.com/73802219/98742537-f5a12000-23f1-11eb-85ee-ca4b7031fcbc.png)

![image](https://user-images.githubusercontent.com/73802219/98742564-02be0f00-23f2-11eb-9f2e-3ed1e00c758c.png)

![image](https://user-images.githubusercontent.com/73802219/98742625-19646600-23f2-11eb-862a-b2b3eea45bdc.png)
